modules/react/button/stories/button/examples/Secondary.tsx

```diff
@@ -4,11 +4,18 @@ import {SecondaryButton} from '@workday/canvas-kit-react/button';
 import {HStack} from '@workday/canvas-kit-react/layout';
 import {plusIcon} from '@workday/canvas-system-icons-web';
 
-export const Secondary = () => (
-  <HStack spacing="s" padding="s">
-    <SecondaryButton>Secondary</SecondaryButton>
-    <SecondaryButton icon={plusIcon} iconPosition="right">
-      Secondary
-    </SecondaryButton>
-  </HStack>
+export const Secondary = props => (
+  <>
+    <HStack spacing="s" padding="s">
+      <SecondaryButton {...props}>Secondary</SecondaryButton>
+      <p>{JSON.stringify(props.theme)}</p>
+      <SecondaryButton icon={plusIcon} iconPosition="start" {...props}>
+        Secondary
+      </SecondaryButton>
+      <SecondaryButton icon={plusIcon} iconPosition="end" {...props}>
+        Secondary
+      </SecondaryButton>
+      <SecondaryButton icon={plusIcon} {...props} />
+    </HStack>
+  </>
```

Comment:
This is weird. @RayRedGoose has definitely updated this in another PR that we merged to master. Maybe it hasn't been merged into pre-prerelease/major? This will blow up Gatsby / the docs site. What's below will fix it.

```suggestion
export const Secondary = () => (
    <HStack spacing="s" padding="s">
      <SecondaryButton>Secondary</SecondaryButton>
      <SecondaryButton icon={plusIcon} iconPosition="start">
        Secondary
      </SecondaryButton>
      <SecondaryButton icon={plusIcon} iconPosition="end">
        Secondary
      </SecondaryButton>
      <SecondaryButton icon={plusIcon} />
    </HStack>
```